core/help/help.py

```diff
@@ -24,6 +25,23 @@
 )
 
 ctx = Context()
+
+def update_lists():
+    mapping = {}
+    for name in registry.lists.keys():
+        list_name = name.split('.')[-1]
+        display_name = list_name.replace("_", " ")
+
+        short_names = actions.user.create_spoken_forms(
+            display_name,
+            generate_subsequences=False,
+        )
```

Comment:
I copied this from somewhere else in the file, I'm not sure exactly what it's doing, as the output is usually the same as the input, other than abbreviations.

In any case, I'm not sure exactly what's being done (if anything) to convert context names to name that can be pronounced for `user.help_contexts`, but ideally we want something like that here.